core/ecschema-rpc/common/src/ECSchemaRpcInterface.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelRpcProps, RpcInterface, RpcManager } from "@bentley/imodeljs-common";
+import { SchemaKey } from "@bentley/ecschema-metadata";
+
+/***
+ * Defines an RPC interface to get schema information from a given iModel context.
+ * Method @see getSchemaNames will return the names of schemas that live in this iModel.
+ * The actual schemas can be downloaded using @see getSchemaJSON to get the schema as JSON props
+ * or @see getSchemaXml to get the schemas as XML document.
+ * @Internal
```

Comment:
lowercase `internal`, not sure if it makes a difference or not but that's the pattern everywhere else.